components/Reviews/Stars.tsx

```diff
@@ -7,7 +7,7 @@ function Star(props: Star) {
 		<svg
 			xmlns='http://www.w3.org/2000/svg'
 			className='icon icon-tabler icon-tabler-star-filled'
-			width={24}
+			width={18}
```

Comment:
you can remove width and height and include it for parent, after that you can easily change size in other components